### PR TITLE
Update acpid

### PR DESCRIPTION
**Pull request by nobodysu**
```
debian11$ grep -E '^\s+/dev/null' /etc/apparmor.d/abstractions/base
  /dev/null                      rw,
debian11$ find /usr/*bin -name acpid
/usr/sbin/acpid
```

```
ubuntu-lts$ grep -E '^\s+/dev/null' /etc/apparmor.d/abstractions/base
  /dev/null                      rw,
ubuntu-lts$ find /usr/*bin -name acpid
/usr/bin/acpid
```

### CODE REVIEW

**Comment by roddhjav** [comment]
Thank a lot for this PR. And good work too. This profile was missing a bit of love.

Can you just apply the profile guideline to it.

File: apparmor.d/profiles-a-f/acpid [hunk 1]
```diff
@@ -20,17 +20,68 @@ profile acpid @{exec_path} flags=(attach_disconnected) {
...
+
+  /etc/acpi/powerbtn-acpi-support.sh r,
+
+  /{usr/,}bin/sed          rix,
+  /{usr/,}bin/pgrep        rix,
+  /{usr/,}bin/{,e}grep     rix,
+  /{usr/,}bin/pinky        rix,
+  /{usr/,}bin/{,ba,da}sh   rix,
+  /{usr/,}bin/dbus-send    rix,
+  /{usr/,}bin/systemctl    rix,
```
Comment (roddhjav) [inline apparmor.d/profiles-a-f/acpid:57]: Here you should use `/{usr/,}bin/systemctl rPx -> child-systemctl, instead

File: apparmor.d/profiles-a-f/acpid [hunk 2]
```diff
@@ -20,17 +20,68 @@ profile acpid @{exec_path} flags=(attach_disconnected) {
...
+  /{usr/,}sbin/killall5    rix,
```
Comment (roddhjav) [inline apparmor.d/profiles-a-f/acpid:58]: Some distributions do not have sbin directory (eg: arch). So you should use `{s,}bin`.

**Comment by roddhjav** [comment]
Thank for the update, it is better now, once you fix the minor issue I can merge it.

**Comment by nobodysu** [comment]
Done.

File: apparmor.d/profiles-a-f/acpid [hunk 3]
```diff
@@ -17,20 +17,63 @@ profile acpid @{exec_path} flags=(attach_disconnected) {
 
   @{exec_path} mr,
 
-  /{usr/,}bin/{,ba,da}sh        rix,
+  /{usr/,}bin/{ba,da,}sh        rix,
   /{usr/,}bin/logger            rix,
 
+  /etc/acpi/powerbtn-acpi-support.sh rPx -> acpid//powerbtn-acpi-support.sh,
```
Comment (nobodysu) [inline apparmor.d/profiles-a-f/acpid:23]: Double slash or underscore here? Slash will collide with `rCx` visually.

**Pull request merged**